store/postgres/src/relational_queries.rs

```diff
@@ -932,22 +932,22 @@ impl<'a> QueryFragment<Pg> for RevertClampQuery<'a> {
         //   update table
         //     set block_range = int4range(lower(block_range), null)
         //   where block_range @> $block
-        //     and not upper_inf(block_range)
+        //     and not block_range @> INTMAX
         //   returning id
         out.push_sql("update ");
         out.push_identifier(&self.schema)?;
         out.push_sql(".");
         out.push_identifier(self.table.name.as_str())?;
         out.push_sql("\n   set ");
-        out.push_identifier(BLOCK_RANGE)?;
+        out.push_identifier(BLOCK_RANGE_COLUMN)?;
         out.push_sql(" = int4range(lower(");
-        out.push_identifier(BLOCK_RANGE)?;
+        out.push_identifier(BLOCK_RANGE_COLUMN)?;
         out.push_sql("), null)\n where");
-        out.push_identifier(BLOCK_RANGE)?;
+        out.push_identifier(BLOCK_RANGE_COLUMN)?;
         out.push_sql(" @> ");
         out.push_bind_param::<Integer, _>(&self.block)?;
-        out.push_sql(" and not upper_inf(");
-        out.push_identifier(BLOCK_RANGE)?;
+        out.push_sql(" and not (");
```

Comment:
Are the parenthesis still required/desired here?

Reply:
Removed them. Commit 5dc5aa08